harpy/convert.py

```
@@ -454,6 +362,101 @@ def standardize(sam, quiet):
+            gzip = subprocess.Popen(["gzip"], stdin = subprocess.PIPE, stdout = out_fq)
+            for record in in_fq:
+                _bx = bx_search(record)
+                if not _bx:
+                    inline_bc = "N"*18
+                    inline_qual = "I"*18
+                else:
+                    if NUCLEOTIDE_FMT:
+                        inline_bc = _bx
+                        inline_qual = "I"*len(_bx)
+                    else:
+                        nuc_bx = bc_inventory.get(_bx, None)
```

Ruff (0.11.9)

Use `bc_inventory.get(_bx)` instead of `bc_inventory.get(_bx, None)`

Replace `bc_inventory.get(_bx, None)` with `bc_inventory.get(_bx)`

(SIM910)